Models/Controller/controller_mosaik.py

```diff
@@ -9,7 +9,7 @@
     import Models.Controller.controller_model as controller_model
 #import Battery.model as batterymodel
 import sys
-sys.path.insert(1,'/home/illuminator/Desktop/Final_illuminator')
+sys.path.insert(1, '/home/illuminator/Desktop/Final_illuminator')

```

Comment:
Avoid using absolute paths. This will throw an error in someone else computer.

Reply:
I know, but I expected you guys to update this in the new version. The only change I made here was to add a space when auto-formatting the file for readability.

Reply:
Alright